internal/resolver/dns/dns_resolver_test.go

```diff
@@ -662,13 +675,22 @@ func txtLookup(host string) ([]string, error) {
 
 func TestResolve(t *testing.T) {
 	testDNSResolver(t)
+	testDNSResolverExponentialBackoff(t)
```

Comment:
Any reason not to make this its own top-level test case?

Reply:
Changed.